tests/testcafe/tests.js

```diff
@@ -332,3 +366,7 @@ publicLibraryButtons('Master Button',      0, 'eb19dffdb38641d5556e5fdb2c47c62b'
   'masterbutton', 'redbutton', 'orangebutton', 'yellowbutton', 'greenbutton', 'bluebutton', 'indigobutton',
   'violetbutton', 'fae4', 'vbx5'
 ]);
+
+hiddenLibraryButtons('Test_Room',     0, '44784f8e8957d15535b7bf04ea9a72c3', [
+  'clickThis', 'button1', 'button2', 'type', 'button3', 'button4' 
```

Comment:
I don't see a widget with ID `type`. Either you change it in a routine somewhere or you just messed up when editing the button (it has `type: "type"`).

Reply:
![image](https://user-images.githubusercontent.com/78324099/129882389-954b17a1-01ab-4f89-a594-ce28aea7d86d.png)
It has a type and id of "type".  However, I'm waiting for an update to the library file.  I made a more simplified version that passes on my home server.